_build-and-test-all.sh

```diff
@@ -32,4 +33,16 @@ echo 'show dbs' |  ./mongodb-cli.sh -i
 
 ./gradlew :end-to-end-tests:cleanTest :end-to-end-tests:test
 
+cat "add-database-id-support-to-eventuate-mysql.migration.sql" | ./mysql-cli.sh -i
```

Comment:
Shouldn't this only be executed when DATABASE == mysql?

Reply:
yes, I was thinking to add all conditions when we discuss mysql, but ok, I will do it now